experimenter/experimenter/features/manifests/nimbus_fml_loader.py

```diff
@@ -68,5 +80,55 @@ def get_minor_version_ref(self, version):
             patch=version.micro,
         )
 
-    def create_client(self, path: str, channel: str, ref: str) -> FmlClient:
-        return FmlClient.new_with_ref(path, channel, ref)
+    # Todo: Add versioning https://mozilla-hub.atlassian.net/browse/EXP-3875
+    def _get_fml_clients(self, versions: list[str]) -> list[FmlClient]:
+        refs = self._get_version_refs(versions)
+        file_path = self._get_local_file_path()
+        clients = []
+        for r in refs:
+            client = self._create_client(str(file_path), self.channel)
+            if client is not None:
+                clients.append(client)
+            else:
+                logger.info(
+                    "Nimbus FML Loader: "
+                    f"Failed to create FML clients for file path {file_path}, "
+                    f"channel {self.channel}, and ref {r}."
+                )
+        return clients
+
+    def _create_client(self, path: str, channel: str) -> FmlClient:
+        return FmlClient(path, channel)
+
+    def _get_inspectors(
+        self, clients: list[FmlClient], feature_id: str
+    ) -> list[FmlFeatureInspector]:
+        if inspectors := [client.get_feature_inspector(feature_id) for client in clients]:
```

Comment:
Looking at the UDL, it looks like it can return None https://github.com/mozilla/application-services/blob/d58715194a98867372cdce8f75bd32b2e68b953b/components/support/nimbus-fml/src/fml.udl#L72